fuel-vm/src/interpreter/contract.rs

```diff
@@ -196,8 +230,12 @@ struct TransferCtx<'vm, S, Tx> {
     context: &'vm Context,
     balances: &'vm mut RuntimeBalances,
     receipts: &'vm mut ReceiptsCtx,
+    profiler: &'vm mut Profiler,
```

Comment:
You forgot to update `transfer_output`

Reply:
I though TRO doesn't create markle tree entries? Am I missing something here?